### PR TITLE
Makes Xenobiology Simpler. Removes Slimes

### PR DESCRIPTION
**Pull request by Fox-McCloud**
and adds simple mob slimes

😛 

From TG.

This is mostly just a refactor; there's not really all that many changes here; this just kills off stupid carbon based slimes and converts them over to simple mobs, as they should be.

Slimes now have a HUD and actions buttons; the HUD is pretty simple; it's really just a health icon

fixes: https://github.com/ParadiseSS13/Paradise/issues/11977

Actual changes:
- Slimes use action buttons instead of verbs
- Using a Docility potion on a slime that's rabid will make it not rabid (but not docile)
- Using bloodlust potion on a slime that's docile will make it not docile (but not rabid)

:cl: Fox McCloud
tweak: Slimes are a simple mob subtype
add: using a bloodlust slime potion on a docile slime removes the docility
add: using a docility potion on a rabid slime calms them down
tweak: slime core removal surgery steps are now just scalpel->hemostat
add: Slimes have action buttons to feed on, evolve, or reproduce
fix: Fixes beepsky arresting slimes (maybe it should be a feature....)
/:cl:

### CODE REVIEW

File: code/game/machinery/adv_med.dm [hunk 1]
```diff
@@ -77,10 +77,9 @@
 		if(occupant)
 			to_chat(user, "<span class='notice'>The scanner is already occupied!</span>")
 			return
-		for(var/mob/living/carbon/slime/M in range(1, TYPECAST_YOUR_SHIT.affecting))
-			if(M.Victim == TYPECAST_YOUR_SHIT.affecting)
-				to_chat(user, "<span class='danger'>[TYPECAST_YOUR_SHIT.affecting.name] has a fucking slime attached to [TYPECAST_YOUR_SHIT.affecting.p_them()], deal with that first.</span>")
-				return
+		if(TYPECAST_YOUR_SHIT.affecting.has_buckled_mobs()) //mob attached to us
+			to_chat(user, "<span class='warning'>[TYPECAST_YOUR_SHIT.affecting] will not fit into [TYPECAST_YOUR_SHIT.affecting] because [TYPECAST_YOUR_SHIT.affecting.p_they()] [TYPECAST_YOUR_SHIT.affecting.p_have()] a fucking slime latched onto [TYPECAST_YOUR_SHIT.affecting.p_their()] head.</span>")
```
Comment (Fox-McCloud) [inline code/game/machinery/adv_med.dm]: `into [src]`

**Comment by MRgaty** [comment]
Newcrit for xenobio slimes when

**Comment by AffectedArc07** [comment]
Title yes

**Comment by Kyep** [comment]
> 
> 
> slimes...were not simple mobs? *scream
> 
> Does this mean white terrors can no longer infect them?

I already have a check in the white terror code that stops them infecting slimes.

**Comment by datlo** [comment]
This PR will most likely fix #11977

**Comment by Fox-McCloud** [comment]
@datlo number one reason to not do this refactor.

**Pull request merged**